### PR TITLE
Basic pyside6 support

### PR DESCRIPTION
**Pull request by corranwebster**
This makes a minimal set of changes to allow the test suite and most examples to run under PySide6.

### CODE REVIEW

**Comment by corranwebster** [comment]
Closing, as it seems to have pulled in some experimental code.

File: pyface/qt/QtOpenGL.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     from PyQt5.QtOpenGL import *
 
 elif qt_api == "pyqt6":
-    from PyQt6.QtOpenGL import *
+    from PyQt6.QtGui import *
```
Comment (rahulporuri) [inline pyface/qt/QtOpenGL.py:19]: Is this change necessary? I still see a `PyQt6.QtOpenGL` subpackage.

**Review by rahulporuri — APPROVED** [review]
LGTM with one question

File: pyface/qt/QtOpenGL.py (hunk 1 above)
Comment (corranwebster) [reply pyface/qt/QtOpenGL.py:19]: That looks like an error - there may be something we need to do for Qt5 and 6 because the content of QtOpenGL is different between the two - but it can be in a different PR if needed.

Comment (corranwebster) [reply pyface/qt/QtOpenGL.py:19]:
```suggestion
    from PyQt6.QtOpenGL import *
```
